docs/deploy/admin.md

```diff
@@ -0,0 +1,93 @@
+# Administration
+
+## Org Import & Export
+
+This guide covers exporting an organization from Browsertrix Cloud and optionally importing it into another Browsertrix Cloud cluster.
+
+Both export and import are two-step processes, which involve:
+
+1. Copying an organization's crawl, upload, and profile files in object storage
+2. Copying an organization's database information via a portable JSON file
+
+### Export
+
+#### Export files
+
+An organization's files are co-located within a "directory" in the S3 bucket being used for storage. This makes it possible to recursively copy all of the files in their original logical structure using tools such as the `aws s3` command-line interface or `rclone`, e.g.:
+
+```
+aws s3 cp s3://current-bucket/<org-id> /path/to/local/directory/<org-id> --recursive --endpoint=https://ams3.digitaloceanspaces.com
+```
+
+It is important to retain the directory structure if you wish to re-import your files into another Browsertrix Cloud cluster later, as some assets such as browser profiles and uploads  have "subdirectory" prefixes.
+
+??? info "Object storage"
```

Comment:
[Info Admonitions](https://docs.browsertrix.cloud/develop/docs/#admonitions) are always used with a conditional to represent the context under which the enclosed information is useful.  This might be a question or perhaps just a regular note?

Reply:
Ah I think a note would be a better fit, thank you!